src/hooks/auth.ts

```diff
@@ -41,24 +41,19 @@ export const useAuth = ({ middleware }: { middleware?: any } = {}) => {
   const csrf = () => api.get('/sanctum/csrf-cookie');
 
   // login
-  const login = async ({ setErrors, ...props }: loginProps) => {
-    setErrors([]);
-
+  const login = async ({ ...props }: loginProps) => {
     await csrf();
 
-    await api
-      .post('/api/login', props)
-      .then(async (resp) => {
-        {
-          console.log(resp.data, '--> resp.data');
-        }
+    try {
+      await api.post('/api/login', props).then(async (resp) => {
         setAuthToken(resp.data);
         await mutate();
         router.push('/campaign');
-      })
-      .catch((error) => {
-        // Handle error
+        toast.success('Login successful');
```

Comment:
no need to show a toast in this case I believe. They already see the dashboard so that already tells that they have logged in.

Reply:
Make sense. Removed this.